features/org.wso2.identity.apps.authentication.portal.server.feature/resources/web.xml.j2

```diff
@@ -125,6 +125,17 @@
         <param-value>true</param-value>
     </context-param>
 
+    <!-- *************** Authentication Portal Custom Configurations ********************** -->
+    {% if authenticationendpoint_custom_properties is defined %}
+    {% for key, value in authenticationendpoint_custom_properties.items() %}
```

Comment:
`authenticationendpoint_custom_properties` -> `authenticationendpoint_context_params` WDYT?

Reply:
fixed. Config changed to `authenticationendpoint.context_params` as discussed